invoke/runners.py

```diff
@@ -476,7 +490,7 @@ def start(self, command):
             )
 
     def default_encoding(self):
-        return locale.getpreferredencoding(False)
+        return locale.getpreferredencoding()
```

Comment:
If I recall, ```False``` is supplied here because that's what the Python IO stack does when setting the encoding. I'd be careful making this change, it's possible it is just trading failures on one platform for failures on another.

I can't offer any specifics, I'm afraid, but I would strongly recommend (a) clearly understanding what difference switching from ```True``` (the default) to ```False``` makes, and (b) adding a comment clarifying the logic, so that we don't later get someone switching back to ```True``` because that works better on their platform.